addons/binding/org.openhab.binding.windcentrale/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,22 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Windcentrale Binding
+Bundle-SymbolicName: org.openhab.binding.windcentrale;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: com.google.common.collect,
+ com.google.gson,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.types,
+ org.slf4j
+Service-Component: OSGI-INF/*
+Export-Package: org.openhab.binding.windcentrale,
+ org.openhab.binding.windcentrale.handler
+Require-Bundle: org.apache.commons.io
```

Comment:
The manifest must not contain any “Require-Bundle” entries. Instead, “Import-Package” must be used.